src/js/shot-generator/BonesHelper.js

```diff
@@ -251,14 +251,15 @@ function filter_array(test_array) {
     return result
 }
 
-function BonesHelper( object, object3D ) {
+function BonesHelper( object, object3D, weirdoFBMesh ) {
```

Comment:
Hmm. Could be more descriptive by using an options object to describe alterations. Maybe something like:

`BonesHelper( object, object3D, { boneLengthScale = 1 })`

And if we detect the Facebook exporter issue, pass an argument of `{ boneLengthScale: 100 }`